src/iocmme.cpp

```diff
@@ -423,6 +428,27 @@ void CmmeInput::CreateBarline(pugi::xml_node barlineNode)
     m_currentContainer->AddChild(barLine);
 }
 
+void CmmeInput::CreateChord(pugi::xml_node chordNode)
+{
+    assert(m_currentContainer);
+
+    Chord *chord = new Chord();
```

Comment:
What about the chord duration? I guess in CMME it is given in the notes (each of them). The practice in Verovio is to expect the longest duration to be given in the chord

Reply:
I *think* my latest commit does this, but please check